### DISCUSSION TITLE
How to read all messages from a group?

### DISCUSSION THREAD

**Discussion by Teslestien** [python-telegram-bot/python-telegram-bot#2780] [Q&A] [answered]
Hi, I was trying to make a bot to filter out inappropriate messages in my group. I wanted to see all messages sent into the group even the ones that didn't start with `/`. How can I do so?

**Comment by zeroone2numeral2** [comment, marked as answer]
From the bot API docs: [changing a bot's privacy mode](https://core.telegram.org/bots#privacy-mode)

**Reply by Teslestien** [reply]
> From the bot API docs: [changing a bot's privacy mode](https://core.telegram.org/bots#privacy-mode)

I found that before but it's not what I was searching for. I meant to say how to read messages sent to the group. anyways, I found it it was this : `update.message.text`. I just wanted to see the messages sent by the users in the group.
